service/discovery/azure/storage.go

```diff
@@ -263,15 +263,16 @@ func getBlockStorageAtRestEncryption(disk compute.Disk) voc.HasAtRestEncryption
 
 	if disk.Encryption.Type == compute.EncryptionAtRestWithPlatformKey {
 		enc = voc.ManagedKeyEncryption{AtRestEncryption: &voc.AtRestEncryption{
-			Algorithm: "", // not available
+			Algorithm: "AES256", // TODO(garuppel): I changed it due to https://docs.microsoft.com/en-us/azure/storage/common/storage-service-encryption
 			Enabled:   true,
 		}}
-	} else if disk.Encryption.Type == compute.EncryptionAtRestWithCustomerKey && *disk.EncryptionSettingsCollection.Enabled {
+		// TODO(garuppel): Check the changed code (commented-out part and removed condition in if statement)
```

Comment:
I do not understand why you commented it out. We need the SourceVault-ID. As it is now, the the keyUrl is always empty.

I checked both conditions, because you can also turn off encryption and, as I understand, the Encryption.Type still says 'EncryptionAtRestWithPlatformKey'/ 'EncryptionAtRestWithCustomerKey' or 'EncryptionAtRestWithPlatformAndCustomerKeys'. [https://docs.microsoft.com/en-us/azure/templates/microsoft.compute/disks?tabs=json#encryptionsettingscollection](url)